week1/hackyourtemperature/server.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+app.get('/', (req, res) => {
+  res.send('hello from backend to frontend!');
+});
+app.post('/weather', (req, res) => {
+  const cityName = req.body.cityName;
```

Comment:
You can destructure it directly 

```suggestion
  const { cityName } = req.body;
```